include/pacemaker.h

```diff
@@ -542,6 +542,8 @@ int pcmk_ticket_set_attr(xmlNodePtr *xml, const char *ticket_id, GHashTable *att
  */
 int pcmk_ticket_state(xmlNodePtr *xml, const char *ticket_id);
 
+int pcmk_update_configured_schema(xmlNode **xml);
+
```

Comment:
Actually this doesn't need to be in libpacemaker, it can just stay in libcrmcommon (include/crm/common/schemas.h)

Reply:
and this one needs to be dropped